docs/contributing.rst

```diff
@@ -54,6 +54,9 @@ in pull request names are enumerated :ref:`in the release notes documentation<re
 Pull requests that change the library's public API require a :ref:`release note<release_notes>`.
 If your pull request doesn't change the public API, apply the ``no-changelog`` label.
 
+Once approved, pull requests should be merged with the "Squah and Merge" option.
```

Comment:
Take a look at the spellchecker error here https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/53936/workflows/f337def8-83a6-4dba-8994-968bb0b27174/jobs/3434031
```suggestion
Once approved, pull requests should be merged with the "Squash and Merge" option.
```